app/models/apple/podcast_container.rb

```diff
@@ -133,18 +131,14 @@ def self.upsert_podcast_container(episode, row)
     end
 
     def self.get_podcast_containers_via_episodes(api, episodes)
-      # Only query for episodes that don't have a podcast container
-      # The container. Assume that if we have a container record, we don't need to poll.
-      eps_without_container = episodes.filter { |ep| ep.apple_persisted? && ep.missing_container? }
-
```

Comment:
Trying to move special guards and state checks into more top level concerns (like `PodcastContainer#delivery_settled?`)